ansible/roles/slurm_node/templates/01_slurm.conf.j2

```diff
@@ -0,0 +1,2 @@
+[domain/ikim.uk-essen.de]
```

Comment:
```suggestion
[domain/{{ groups['slurm_nodes'][0].split('.', 1)[1] }}]
```
Could work as well, assuming that the entry in the inventory is in the form [host].[domain]